src/Tizen.NUI/src/public/Rendering/Renderer.cs

```diff
@@ -678,6 +675,25 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
             Interop.Renderer.DeleteRenderer(swigCPtr);
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void Dispose(bool disposing)
+        {
+            if (disposed)
+            {
+                return;
+            }
+
+            if (disposing)
+            {
+                // dispose managed state (managed objects).
+                CurrentGeometry?.Dispose();
+                CurrentShader?.Dispose();
+            }
+
+            // Call the base class implementation.
+            base.Dispose(disposing);
+        }
+
```

Comment:
우선, https://github.sec.samsung.net/dotnet/NUIBackend/issues/34#issuecomment-1219215 
여기에서 언급했듯이, BaseHandle을 상속한 객체는 disposing이 false인 상태에서 disposing이 true로 호출되는 상황이 발생합니다. 
따라서, 차라리 IDisposable 객체에 대한 dispose는  Dispose(DisposeTypes type) 여기를 통해서 하는것이 안전할것 같습니다.
```suggestion
        protected override void Dispose(DisposeTypes type)
        {
            if (type == DisposeTypes.Explicit)
            {
                CurrentGeometry?.Dispose();
                CurrentShader?.Dispose();
            }
            base.Dispose(type);
        }
```

Reply:
네. 맞습니다. 수정해 주셔서 감사합니다. 반영 하였습니다.